src/helpers/get-ogc-items-stream.ts

```diff
@@ -1,27 +1,61 @@
 import { PagingStream } from '../paging-stream';
 import { getPagingStream } from './get-paging-stream';
 import * as _ from 'lodash';
+import axios from 'axios';
```

Comment:
Should we eventually switch to native fetch instead?

Reply:
I have added to do comment to use native fetch. Will create ticket for dev backlog.